reactioncommerce/reaction
# Fix PDP images not associated with variant/options

This fixes #2647 

# Steps to test
1. As admin, select a variant option in PDP
1. Click on the thumbnails to upload image for the variant.
<img width="986" alt="basic_reaction_product" src="https://user-images.githubusercontent.com/25609312/29516063-dde97c68-8666-11e7-8556-3477eb7acadc.png">

1. Observe that the image is attached to the selected variant. And if you select another option, you should not see the image
<img width="987" alt="basic_reaction_product2" src="https://user-images.githubusercontent.com/25609312/29516351-f6b90af0-8667-11e7-835f-8e606d51000f.png">
